Detections/ThreatIntelligenceIndicator/IPEntity_AzureFirewall.yaml

```diff
@@ -0,0 +1,63 @@
+id: 0b904747-1336-4363-8d84-df2710bfe5e7
```

Comment:
Added in the id for you.  You can use new-guid in PowerShell to generate your own...or use any GUID generator in the future. :)

Reply:
Thank you very much for explaining me.